src-tauri/tauri.conf.json

```diff
@@ -15,7 +15,8 @@
   "tauri": {
     "allowlist": {
       "shell": {
-        "sidecar": true
+        "sidecar": true,
+        "open": ".*"
```

Comment:
FYI, this is the only actual change. Tauri will add any rust dependencies needed, which in turn updates the lockfile.